code/modules/mining/ores_coins.dm

```diff
@@ -471,7 +509,7 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 						"uranium" = /obj/item/clothing/gloves/ring/uranium)
 	var/typekey = typelist[cmineral]
 	if(ispath(typekey))
-		to_chat(user, "<span class='notice'>You make [src] into a ring.</span>")
+		to_chat(user, "<span class='notice'>You carefully cut a hole into [src] turning it into a ring.</span>")
 		new typekey(get_turf(loc))
```

Comment:
Completely optional, but you can use a structure like this to make it put the ring in your hand instead of dropping it on the ground
```
	var/obj/item/clothing/gloves/ring/ring = new typekey()
	qdel(src)
	user.put_in_hands(ring)
```
Again, it's optional but still a small QOL improvement